Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Arthas' Event Chain

## Changelog:
- When the Plague of Lordaeron starts, the king of the targeted kingdom (Terenas) may send his trustworthy commander (Arthas) to investigate the outbreak. Arthas may invite a player if Arthas has more than 0 opinion of them. The main characters are:
  - Arthas
  - Terenas
  - Player (generic Terenas' vassal)
- When you're in the Arthas' expedition you wear armor.

# Design Document
#777

# Tasks:
- [x] Set intervals between events
  - Check WC3 missions for dates
  - [x] Adjust history to fit it
- [x] Change activity icon
- [x] Change scenes of plague events to new ones
- [x] Add Sadistic to Stratholme
- [x] Replace liege by leader
- [x] Arthas wears hood in the last events
- [x] Stop event chain if Ner'zhul dies
- [x] Add an alternative description if Hearthglen wasn't found
- [x] Finish `on_trigger_fail` if Stratholme wasn't found
- [x] Add Lordaeron throne room
- [x] Arthas shouldn't wear a hood after he becomes ruler
- [x] Fix no ship event
- [x] Arthas stays king and holds the capital


## Tests:


# How to test:
@Warcraft-GoA-Development-Team/ck3-testers play Terenas, Arthas or generic Terenas' vassal and try different options in this event chain.

@Warcraft-GoA-Development-Team/english-proofreaders open the [Page](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/788/files?authenticity_token=HdCBWFbgecjuZQBS%2FFmFv%2Fjci4F9SxwA%2FBzNkYAM%2BnVZ3ECtgHlsfJ1595UkOcfdTtKLFr%2BkoNBFoKu5vbStXg%3D%3D&file-filters%5B%5D=.yml), scroll to the bottom to the `.yml` files, check them, suggest changes. If you forgot, here's the [Suggestion Guidelines](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/How-to-be-Tester-(For-CK3)#reviewing-code).
